specification/trace/semantic_conventions/messaging.md

```diff
@@ -80,6 +80,38 @@ Often such destinations are unnamed or have an auto-generated name.
 
 Given these definitions, the remainder of this section describes the semantic conventions for Spans describing interactions with messaging systems.
 
+### Context propagation
```

Comment:
I added this new section at the top, since I believe this is an important topic for whoever is consuming this document.